test/blackbox-tests/sanitize.ml

```diff
@@ -0,0 +1,31 @@
+open Printf
```

Comment:
We can do this in `cram.mll` directly. We already sanitise a few things there such as OS-specific extensions.